packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -761,6 +762,13 @@ class RawScrollbar extends StatefulWidget {
   /// Cannot be null, defaults to [Duration.zero].
   final Duration pressDuration;
 
+  /// A check that specifies whether a [ScrollNotification] should be
+  /// handled by this widget.
+  ///
+  /// By default, checks whether `notification.depth == 0`. Set it to something
+  /// else for more complicated layouts.
```

Comment:
Let's make this documentation a little clearer. Often I look to the bugs I am fixes that describe the problem to help write the docs. 
Mentioning how this can be helpful when the scrollbar is wrapped around multiple or nested scroll views would be a good isea, since that appears to be the most common issue you're solving here.